<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>New_Cooker</class>
 <widget class="QDialog" name="New_Cooker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#New_Cooker { 
	background-color: rgb(255, 255, 255);
} 
#New_Cooker * { 
border-image:url(); 
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="NameLb">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="NameLe"/>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="AgeLe"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Group</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="GroupLe"/>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="BackBtn">
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="AfrBtn">
     <property name="text">
      <string>Submit</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Age</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
